src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/security-recommendations/SecurityRecommendationsLayout.tsx

```diff
@@ -70,9 +70,12 @@ export function SecurityRecommendationsLayout(
             variant="primary"
             small
             // TODO: Add test once MNTOR-1700 logic is added
-            /* c8 ignore next 3 */
+            /* c8 ignore next 6 */
             onPress={() => {
-              // TODO: MNTOR-1700 Add routing logic
+              location.href = getNextGuidedStep(
+                props.data,
+                stepMap[props.type],
+              ).href;
             }}
```

Comment:
I think this can just be a link?

```suggestion
            href={
              getNextGuidedStep(
                props.data,
                stepMap[props.type],
              ).href
            }
```